src/plugins/presentation_util/public/index.ts

```diff
@@ -11,6 +11,8 @@ import { pluginServices } from './services';
 
 export { PresentationUtilPluginSetup, PresentationUtilPluginStart } from './types';
 
+export { experimentIDs, Experiment, ExperimentID } from '../common';
```

Comment:
I resolved this in the parent PR, thankfully.  Let me know if you have other instances where things didn't line up.